src/js/report.js

```diff
@@ -24,16 +24,19 @@ function Report() {
 
 Report.prototype = {
   open: function() {
-    document.getElementById('report-link').href = this.linkToChromiumBug_();
-    document.getElementById('report-dialog').open();
+    document.querySelector('report-dialog').open();
   },
 
-  downloadReport: function() {
-    var content = encodeURIComponent(this.getContent_());
-    var link = document.createElement('a');
-    link.setAttribute('href', 'data:text/plain;charset=utf-8,' + content);
-    link.setAttribute('download', 'testrtc-' + (new Date().toJSON()) + '.log');
-    link.click();
+  // Clears the log and adds sytem-info from the device.
```

Comment:
The user might be uploading more than once... (e.g. try to upload and then later reupload with a slightly different description).... i don't want us to be like: "Oh sorry can you repro again?").